CMSgov/mdct-mcr
# drawer padding fix

## Description

Drawer padding was added if you had 'always show scrollbar' in your settings.
 https://qmacbis.atlassian.net/browse/MDCT-1885

### How to test

Go to any page that has a drawer, open the drawer, set your scroll to 'always' in your system preferences->general ([tutorial](https://www.macinstruct.com/tutorials/set-scroll-bars-to-always-display-on-your-mac/)), check that no extra padding was added.

### Changed Dependencies

n/a

## Code author checklist

## Reviewer checklist (two different people)
